calm/control-example/control-specification.json

```diff
@@ -0,0 +1,46 @@
+{
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "$id": "control-specification.json",
+  "title": "Example Production Control Specification",
+  "properties": {
+    "control-id": {
+      "const": "ci-arch-001"
+    },
+    "name": {
+      "const": "Architecture review pre-production"
```

Comment:
So this gets copied into each system's evidence for this control?  What would happen if one of these const fields were changed (e.g. the scope) - do people have to regenerate all the control evidence files for all the systems?

Reply:
The schema would no longer validate and this is similar to calm where versions would need to be considered.